ndmg/register.py

```diff
@@ -266,6 +267,17 @@ def gen_tissue(self):
         print("Extracting brain from raw T1w image...")
         reg_utils.t1w_skullstrip(self.t1w, self.t1w_brain, self.skull)
 
+        #  QA part of skull strip
+        last_folder = self.namer.dirs["qa"]["base"]
+        QA_skullstrip_path = last_folder + '/skull_strip'
+        folder = os.path.exists(QA_skullstrip_path)
+        if not folder:
+            os.makedirs(QA_skullstrip_path)
+        print('QA_skullstrip_path  ', QA_skullstrip_path)
+        self.namer.dirs["qa"]["skull_strip"] = QA_skullstrip_path
+        gen_overlay_pngs(brain=self.t1w_brain, origional=self.t1w, outdir=self.namer.dirs["qa"]["skull_strip"], loc=0,
```

Comment:
Positional arguments do not have to be directly specified, you can put `gen_overlay_pngs(self.t1w_brain, self.t1w, self.namer.dirs["qa"]["skull_strip"])`

If the value is already specified in the function (like loc, mean, minthr, maxthr, edge) you don't have to define them when you call the function

Reply:
fixed it in latest version

Reply:
![image](https://user-images.githubusercontent.com/55203069/70834718-0bb0f580-1dc9-11ea-966b-f7e13c8710a0.png)